ova/Libraries/elastic_functions.sh

```diff
@@ -0,0 +1,247 @@
+etc_elastic="/etc/elasticsearch"
+etc_filebeat="/etc/filebeat"
+etc_kibana="/etc/kibana"
+usr_kibana="/usr/share/kibana"
+
+set_elastic_repository(){
+
+    rpm --import https://packages.elastic.co/GPG-KEY-elasticsearch
+    echo -e "[elasticsearch-${ELK_MAJOR}.x]\nname=Elasticsearch repository for ${ELK_MAJOR}.x packages\nbaseurl=https://artifacts.elastic.co/packages/${ELK_MAJOR}.x/yum\ngpgcheck=1\ngpgkey=https://artifacts.elastic.co/GPG-KEY-elasticsearch\nenabled=1\nautorefresh=1\ntype=rpm-md" | tee /etc/yum.repos.d/elastic.repo
+
+}
+
+install_elasticsearch(){
+
+    yum install elasticsearch-${ELK_VERSION} -y
+    systemctl daemon-reload
+    systemctl enable elasticsearch.service
+
+}
+install_filebeat_7(){
+
+    yum install filebeat-${ELK_VERSION} -y
+    cp -f ${config_files}/filebeat.yml ${etc_filebeat}/filebeat.yml
+
+    sed -i "s/YOUR_ELASTIC_SERVER_IP/localhost/" ${etc_filebeat}/filebeat.yml
+    chmod go+r ${etc_filebeat}/filebeat.yml
```

Comment:
This line isn't necessary.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/223/commits/06b971366820c9d9cbf359ba4c3b01c427772310